### PR TITLE
update the drone line docs.

### PR DESCRIPTION
**Pull request by appleboy**
as title

### CODE REVIEW

File: content/appleboy/drone-line/index.md [hunk 1]
```diff
@@ -132,22 +131,24 @@ pipeline:
...
++       build {{build.number}} succeeded. Good job.
++     {{else}}
++       build {{build.number}} failed. Fix me please.
++     {{/success}}
 ```
 
-# Parameter Reference
+# Secret Reference
 
 channel_secret
```
Comment (tonglil) [inline content/appleboy/drone-line/index.md]: should this also include `line_channel_secret`?

File: content/appleboy/drone-line/index.md [hunk 2]
```diff
@@ -168,73 +169,3 @@ locations
 
 stickers
 : a vaild sticker format
-
-# Template Reference
```
Comment (tonglil) [inline content/appleboy/drone-line/index.md]: If these are still valid, it should link somewhere for users to reference (like http://plugins.drone.io/drone-plugins/drone-slack/).

File: content/appleboy/drone-line/index.md (hunk 1 above)
Comment (appleboy) [reply content/appleboy/drone-line/index.md]: Done!

Comment (appleboy) [reply content/appleboy/drone-line/index.md]: see https://github.com/drone/drone-plugin-index/pull/112/commits/0ee8253f3d2f8fdcfb2944bd6b53720f667cf6bf

File: content/appleboy/drone-line/index.md (hunk 2 above)
Comment (appleboy) [reply content/appleboy/drone-line/index.md]: Yes. we can merge this PR before document the template lib and I will update the `drone-line` library.

**Comment by tonglil** [comment]
Looking forward to that drone/docs PR @appleboy :)

**Pull request merged**